modules/autotagging-commons/common/src/main/java/org/opensearch/rule/service/IndexStoredRulePersistenceService.java

```diff
@@ -52,9 +54,24 @@
  * @opensearch.experimental
  */
 public class IndexStoredRulePersistenceService implements RulePersistenceService {
+    // Default value for max rules count
+    public static final int MAX_ALLOWED_RULE_COUNT = 10000;
+
+    // max wlm rules setting name
+    public static final String MAX_RULES_COUNT_SETTING_NAME = "wlm.autotagging.max_rules";
+
+    // max wlm rules setting
+    public static final Setting<Integer> MAX_WLM_RULES_SETTING = Setting.intSetting(
+        MAX_RULES_COUNT_SETTING_NAME,
+        MAX_ALLOWED_RULE_COUNT,
+        10,
```

Comment:
I am assuming 10 is default value? Is there a minimum value as well? Also, default value should be slightly higher like 25/50?

Reply:
It is minimum value this setting can assume and `MAX_ALLOWED_RULE_COUNT ` is default max rule count allowed. I will rename the default constant.